static/js/click_handlers.js

```diff
@@ -389,6 +389,11 @@ $(function () {
         $('#logout_form').submit();
     });
 
+    $('body').on('click', '.login_button', function () {
+        $('#login_form').submit();
+        loader.show_loader(login_button);
+    });
+
```

Comment:
This file is only run in the logged-in app; you want to make your changes in `static/js/portico/signup.js`, not here.